<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formLogin</class>
 <widget class="QDialog" name="formLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inicio de sesion</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/d28c7f7b/banco.png</normaloff>:/images/d28c7f7b/banco.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(255, 255, 255);
color: black;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/icons/banco.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Inicio de sesion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_username">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>212</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>alvaro.quezada</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_password">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quezada$alvaro23</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>323</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 190, 111);</string>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_message">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../a1e250ea/icons.qrc"/>
 </resources>
 <connections/>
</ui>
